crates/emath/src/pos2.rs

```diff
@@ -257,11 +257,11 @@ impl Add<Vec2> for Pos2 {
 }
 
 impl Sub for Pos2 {
-    type Output = Vec2;
+    type Output = Self;
```

Comment:
This doesn't make much sense. The difference between two points in space should be a vector, mathematically. It's the reason why these two exist as separate types in the first place.

Reply:
> This doesn't make much sense. The difference between two points in space should be a vector, mathematically. It's the reason why these two exist as separate types in the first place.

Maybe yes. This was automatically requested during the last inspection after the Pull Request.